deploy-application/entrypoint.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -e
+
+namespace=$1
+application=$2
+name=$3
+version=$4
+configuration=$5
+repo=$6
+
+if [ -z "$application" ]; then
+echo application is not defined
+exit 1
+fi
+
+if [ -z "$name" ]; then
+echo name is not defined
+exit 1
+fi
+
+if [ -z "$configuration" ]; then
+configuration=$(pwd)/config.yaml
+fi
+
+token=$(jq  .Token $HOME/.okteto/.token.json -r)
+url=$(jq  .URL $HOME/.okteto/.token.json -r)/graphql
+
+encoded_configuration=$(base64 $configuration)
+
+echo Deploying $namespace/$name via $url
+
+cat << EOF > deploy.payload
+{
+  "query": "mutation {
+    deployApp(
+    chart: \"${application}\", name: \"${name}\", space: \"${namespace}\", version: \"${version}\",repo: \"${repo}\", config: \"${encoded_configuration}\") {
+    id
+    }
+  }"
+}
+EOF
+
+curl -X POST \
```

Comment:
should we have a cli command?

Reply:
Yes. If we want users to interact with this API (or build their own integration) a CLI would be much better than this. But this is a good way to start

Reply:
How would you do it if the PR changes the chart yamls?
I am thinking that we should have an e2e story for developing apps first before adding these actions

Reply:
This is an initial  exploration of what we can do with actions and apps. We don't have a story for that use case (and many others). 

My idea is to have building blocks, and as people start using them, we can keep adding more complex samples and actions.

Reply:
Before exploring actions, we need to solve how to modify apps without CI. I think that OCI compliant charts will help. These actions are problematic because the user faces a dead end when he needs to modify manifests, and it also has a much bigger learning curve than running `kubectl apply`